frontend/src/i18n/locales/no.ts

```diff
@@ -245,6 +245,7 @@ export const Norwegian: LanguageSchema = {
     },
     superAdmin: {
       identifierAttribute: 'Identifier Attribute',
```

Comment:
identitetsattributt?

Reply:
Tja, kan ikke se noen god grunn til å ikke oversette – vi gjør jo det med andre DynamoDB-felt. Såvidt jeg kan se heter attributtet `custom:company` i Azure, så det kan se ut til at det bare er vi som kaller det `identifierAttribute`. Iverksetter oversetting 👷

Reply:
tipper så lenge det er oversatt at det går fint-- om det er uklart vil vel noen si ifra 👍🏼

Reply:
Vi kaller det for identifierAttribute fordi det er verdien fra Azure som identifiserer hvilken organisasjon en bruker tilhører. Hadde vi bare kalt det "company" så kunne det blitt forvirrende. Det er også litt legacy, for før så var Objectnet sin "identifierAttribute" bare "Google", siden vi brukte google innlogging.